lib/chromium/webconsole.js

```diff
@@ -283,6 +293,19 @@ var ChromiumConsoleActor = ActorClass({
       result = response.result;
     }
 
+    // Set $_ for future availability.
+    let lastResult = response.result.objectId;
+    // If not an object, provide an descriptive error to inform the user.
+    if (!lastResult) {
+      let boxed = yield this.rpc.request("Runtime.evaluate", {
+        expression: "Error('Sorry, Valence cannot set $_ to primitive values')",
+      });
+      lastResult = boxed.result.objectId;
+    }
+    yield this.rpc.request("Console.setLastEvaluationResult", {
```

Comment:
I'm seeing an error in canary with this.  This `yield` statement should at least be in a try/catch so a failure here doesn't stop all console evaluation from happening.